test/windows_test.dart

```diff
@@ -425,6 +425,12 @@ main() {
         expect(context.relative(r'a\.\b\..\c.txt'), r'a\c.txt');
       });
 
+      test('is case-insensitive', () {
+        expect(context.relative(r'c:\'), r'..\..');
+        expect(context.relative(r'c:\RoOt'), r'..');
+        expect(context.relative(r'c:\rOoT\pAtH\a'), r'a');
+      });
```

Comment:
Do we have tests for POSIX and URL to verify that they are case sensitive? If not, add some.